toolbox/library/src/OSQP.cpp

```diff
@@ -28,8 +28,6 @@
 
 using namespace blockfactory::core;
 
-const unsigned MaxIterations = 100;
-
```

Comment:
as a comment, as discussed in a meeting, this variable was never used in this block and it was here for copy-and-paste from the qpoases block